Content.Server/Interfaces/GameObjects/Components/Interaction/IDragDrop.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Content.Server.Interfaces.GameObjects.Components.Interaction;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Map;
+
+namespace Content.Server.Interfaces.GameObjects.Components.Interaction
+{
+    /// <summary>
+    /// This interface allows the component's entity to be dragged and dropped by mouse onto another entity and gives it
+    /// behavior when that occurs.
+    /// </summary>
+    public interface IDragDrop
+    {
+        /// <summary>
+        /// Invoked server-side when this component's entity is being dragged and dropped on another.
+        ///
+        /// There is no other server-side drag and drop check other than a range check, so make sure to validate
+        /// if this object can be dropped on the target object!
+        /// </summary>
+        /// <returns>true iff an interaction occurred and no further interaction should
```

Comment:
```suggestion
        /// <returns>true if an interaction occurred and no further interaction should
```

Reply:
not a typo, iff has a specific meaning: https://www.merriam-webster.com/dictionary/iff

Reply:
I'll commit it anyway, probably not many people are familiar with that usage